app/src/pages/evaluators/EvaluatorsFilterProvider.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  createContext,
+  Dispatch,
+  PropsWithChildren,
+  SetStateAction,
+  useContext,
+  useState,
+} from "react";
+import invariant from "tiny-invariant";
+
+import { EvaluatorSort } from "@phoenix/pages/evaluators/__generated__/EvaluatorsTableEvaluatorsQuery.graphql";
+
+export type EvaluatorsFilterContext = {
+  filter: string;
+  setFilter: Dispatch<SetStateAction<string>>;
+  selectedEvaluatorIds: string[];
+  setSelectedEvaluatorIds: Dispatch<SetStateAction<string[]>>;
+  sort: EvaluatorSort | null | undefined;
+  setSort: Dispatch<SetStateAction<EvaluatorSort | null | undefined>>;
+};
+
+export const evaluatorsFilterContext =
+  createContext<EvaluatorsFilterContext | null>(null);
+
+export const EvaluatorsFilterProvider = ({ children }: PropsWithChildren) => {
+  const [filter, setFilter] = useState("");
```

Comment:
given that this state can probably change at a relatively high pace, this means that this context will cause a rerender at a pretty high pace. It's not a huge deal but if this does grow we should consider using zustand instead here because that will let you use sort and filter state from context independantly.